pages/doc/logging_log_browser.md

```diff
@@ -19,8 +19,8 @@ The Logs Browser supports in-depth exploration of your logs. As a user with the
 
 To use the Logs Browser:
 
-1. Configure your application and the log shipper to [send logs to Tanzu Observability](logging_send_logs.html).
-1. In a web browser, log in to your Wavefront instance as user with the **Logs** permission.
+1. Configure your application and the log shipper to [send logs](logging_send_logs.html).
+1. In a web browser, log in to your account as user with the **Logs** permission.
```

Comment:
Maybe something like that?

... log in to your product instance as user with the **Logs** permission.